<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>oblsch</class>
 <widget class="QWidget" name="oblsch">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>400</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>750</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>750</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string notr="true">Obliczenia wysokości schodów</string>
  </property>
  <widget class="QTextBrowser" name="textBrowser">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>731</width>
     <height>181</height>
    </rect>
   </property>
   <property name="cursor" stdset="0">
    <cursorShape>IBeamCursor</cursorShape>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::DefaultContextMenu</enum>
   </property>
   <property name="acceptDrops">
    <bool>true</bool>
   </property>
   <property name="toolTip">
    <string/>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhMultiLine</set>
   </property>
   <property name="tabChangesFocus">
    <bool>false</bool>
   </property>
   <property name="undoRedoEnabled">
    <bool>false</bool>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:7.8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8.25pt;&quot;&gt; &lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:8.25pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:8.25pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="overwriteMode">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>150</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>Zapisz do pliku</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>140</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Oblicz</string>
   </property>
   <property name="autoExclusive">
    <bool>false</bool>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>12</y>
     <width>438</width>
     <height>191</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QRadioButton" name="radio1">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="cursor">
       <cursorShape>ArrowCursor</cursorShape>
      </property>
      <property name="acceptDrops">
       <bool>false</bool>
      </property>
      <property name="whatsThis">
       <string/>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Budynki mieszkalne jednorodzinne oraz mieszkania dwupoziomowe.</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
      <property name="autoRepeat">
       <bool>false</bool>
      </property>
      <property name="autoExclusive">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radio2">
      <property name="text">
       <string>Budynki mieszkalne wielorodzinne oraz budynki użytecznosci publicznej.</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radio3">
      <property name="text">
       <string>Przedszkola i żłobki.</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radio4">
      <property name="text">
       <string>Budynki opieki zdrowotnej.</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radio5">
      <property name="text">
       <string>Garaże wbudowane i wolno stojące oraz budynki usługowe do 10 osob.</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radio6">
      <property name="text">
       <string>Schody do kondygnacji podziemnej.</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>40</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Wysokość kondygnacji:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>40</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>cm</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>80</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Minimalna ilość rozwiązań: </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="textEdit_1">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>40</y>
     <width>39</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>300</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="textEdit_2">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>90</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>3</string>
   </property>
  </widget>
  <zorder>layoutWidget</zorder>
  <zorder>textBrowser</zorder>
  <zorder>pushButton</zorder>
  <zorder>pushButton_2</zorder>
  <zorder>label</zorder>
  <zorder>label_2</zorder>
  <zorder>label_3</zorder>
  <zorder>textEdit_1</zorder>
  <zorder>textEdit_2</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
